code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -33,6 +33,11 @@
 	if(istype(A, /obj/item/weapon/storage) || istype(A, /obj/structure/table) || istype(A, /obj/structure/closet) || istype(A, /obj/item/weapon/reagent_containers) || istype(A, /obj/structure/sink) || istype(A, /obj/structure/janitorialcart))
 		return
 
+	if(istype(A, /mob))
+		user.setClickCooldown(25)

```

Comment:
the default cooldown isnt always suitable. value tuned to not be too much when piling on large quantities of chems. it applies as much to acids as to plant b gone. its a balance thing